tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -221,88 +223,131 @@
+
                     <!-- Repos cloned -->
-                    <StackPanel 
-                        Orientation="Vertical" 
-                        Spacing="20" 
-                        BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}" 
-                        BorderThickness="{ThemeResource TopNavigationViewContentGridBorderThickness}"
-                        Visibility="{x:Bind ViewModel.RepositoriesCloned, Converter={StaticResource CollectionVisibilityConverter}}"
-                        Margin="0,0,0,30">
-                        <TextBlock 
-                            x:Uid="ms-resource:///DevHome.SetupFlow/Resources/SummaryPage_ReposClonedDisplay" 
-                            Style="{ThemeResource BodyStrongTextBlockStyle}" 
-                            Padding="0, 20, 0, 0"/>
-                        <StackPanel>
-                            <StackPanel.Resources>
-                                <StackLayout x:Name="LayoutForRepos" Orientation="Vertical" Spacing="10"/>
-                                <DataTemplate x:Key="repoTemplate" x:DataType="models:RepoViewListItem">
-                                    <Grid>
-                                        <Grid.ColumnDefinitions>
-                                            <ColumnDefinition Width="auto"/>
-                                            <ColumnDefinition Width="*"/>
-                                        </Grid.ColumnDefinitions>
-                                        <FontIcon Grid.Column="0"  Margin="3 0 5 0" FontFamily="{StaticResource DevHomeFluentIcons}" Glyph="{x:Bind IsPrivate, Mode=OneWay, Converter={StaticResource BoolToGlyphConverter}}"/>
-                                        <TextBlock 
-                                            Grid.Column="1"
-                                            Text="{x:Bind RepoName}"
-                                            Margin="5 0 0 0"
-                                            TextWrapping="Wrap"/>
-                                    </Grid>
-                                </DataTemplate>
-                            </StackPanel.Resources>
-                            <ItemsRepeater x:Name="TasksItemsRepeater" ItemsSource="{x:Bind ViewModel.RepositoriesCloned, Mode=OneWay}"
-                               Layout="{StaticResource LayoutForRepos}" 
-                               ItemTemplate="{StaticResource repoTemplate}">
-                            </ItemsRepeater>
-                        </StackPanel>
-                    </StackPanel>
+                    <Grid Padding="0,12" Visibility="{x:Bind ViewModel.RepositoriesCloned, Converter={StaticResource CollectionVisibilityConverter}}">
+                        <Expander>
+                            <Expander.Header>
+                                <TextBlock x:Uid="ms-resource:///DevHome.SetupFlow/Resources/SummaryPage_ReposClonedDisplay" Style="{ThemeResource BodyStrongTextBlockStyle}" Padding="0,12"/>

```

Comment:
I don't think you need `ms-resource:///DevHome.SetupFlow/Resources/` here (and everywhere else in markup). I think it would be different if you were referencing resources outside the current project.

```suggestion
                                <TextBlock x:Uid="SummaryPage_ReposClonedDisplay" Style="{ThemeResource BodyStrongTextBlockStyle}" Padding="0,12"/>

```